ext/pdo_sqlite/tests/pdo_sqlite_createcollation.phpt

```diff
@@ -10,9 +10,7 @@ $db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
 
 $db->query('CREATE TABLE IF NOT EXISTS foobar (id INT AUTO INCREMENT, name TEXT)');
 
-$db->query('INSERT INTO foobar VALUES (NULL, "1")');
-$db->query('INSERT INTO foobar VALUES (NULL, "2")');
-$db->query('INSERT INTO foobar VALUES (NULL, "10")');
+$db->query('INSERT INTO foobar VALUES (NULL, "1"), (NULL, "2"), (NULL, "10")');
 
 $db->sqliteCreateCollation('MYCOLLATE', function($a, $b) { return strnatcmp($a, $b); });
```

Comment:
These table names collide with ``ext/pdo_sqlite/tests/pdo_sqlite_createaggregate.phpt`` and ``ext/pdo_sqlite/tests/pdo_sqlite_createfunction.phpt``, ``ext/pdo_sqlite/tests/pdo_sqlite_createfunction_with_flags.phpt``, and ``ext/pdo_sqlite/tests/pdo_sqlite_lastinsertid.phpt``

Reply:
They are run in-memory (note the connection DSN), that's why I didn't have to touch SQLite tests

Reply:
Ohhhhhhhhhhhhhhhhhhhhhh